src/thx/schema/SimpleSchema.hx

```diff
@@ -66,6 +64,12 @@ class SimpleSchema {
       constAlt("none", None, function(a: Option<A>, b: Option<A>) return a == b)
     ]);
 
+  public static function makeNullable<E, A>(s: Schema<E, A>): Schema<E, Null<A>>
+    return oneOf([
+      alt("value", s, function(a: A) return a, function(a: Null<A>) return Options.ofValue(a)),
+      constAlt("novalue", null, function(a: A, b: A) return a == b)
```

Comment:
My understanding here is that this equality function is always going to compare with the `const` value (`null`) ... so strict equality is fine? You do the same just above in `makeOptional`.